src/spatial/views/report_view.py

```
@@ -0,0 +1,37 @@
+from flask import Flask, request, jsonify
```

Ruff

`flask.Flask` imported but unused

Remove unused import: `flask.Flask`

(F401)